src/include/daos_cont.h

```diff
@@ -55,10 +58,10 @@ typedef struct {
 	uint32_t		ci_redun_fac;
 	/** Number of snapshots */
 	uint32_t		ci_nsnapshots;
-	/** Redundancy level */
-	uint32_t		ci_redun_lvl;
```

Comment:
Is this should be a major API change? @mchaarawi

Reply:
i wouldn't make it a major change because the redun_lvl is something new and never used externally. but my question is why we want to remove it, since we recently added the implementation for it?

Reply:
The idea is that items that are less-frequently changing would be in the properties (such as redun_lvl, and probably redun_fac above too - they are already available as properties), and that this structure could contain more dynamically changing information. For this patch I had at first considered (for implementation simplicity / reduced API impact) to store the metadata times as a container property. However in discussion with @johannlombardi had come to this determination, and in particular that the RF/RL information should not be in the structure. This was convenient to (along with structure pad) provide some space for the new metadata time tracking information while keeping the structure size the same (that we must do in the DAOS-2.x releases).

The ci_redun_fac field can also be removed. Something I am considering but not in this patch is to replace it with the number of open handles on the container. It could be helpful for the use case of listing/filtering containers in a pool and making a decision on which ones are less active than others.